multimodal/tests/unittests/others/test_doc_classification.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
Consider moving this test into folder `others_2` to balance test time.

Reply:
done